meson.build

```diff
@@ -4,7 +4,7 @@ project(
   # Note that the git commit hash cannot be added dynamically here
   # It is dynamically added upon import by versioneer
   # See `numpy/__init__.py`
-  version: '1.24.0.dev0',
+  version: '2.0.0.dev0',
```

Comment:
Seeing this here, do we have to liberate the runtime version for the wheel upload? to rename them?!

Reply:
Liberate? I'm happy with the `2.0.0.dev0-cp39` etc wheel names.

Reply:
xref #24065. There is currently a problem uploading periodic builds: the upload script does not upload if the name alreeady exists. Two solutions, each with their own tradeoffs:
- upload anyway
- add a tag to the uploaded name

Reply:
Just found it, thanks. Resolving here since we should keep the discussion on the issue. there's nothing wrong in this PR.